client/components/Footer.jsx

```diff
@@ -1,121 +1,241 @@
 import React from 'react';
 import styled from 'styled-components';
+import { FaCaretDown, FaGlobe, FaFacebookF, FaTwitter, FaBlog } from 'react-icons/fa';
+import {TiSocialInstagram} from 'react-icons/ti';
+import { IconContext } from 'react-icons';
 
 const FooterBox = styled.div`
-  display: flex;
+  display: block;
   background-color: #2f333d;
 `
 const ColumnBox = styled.div`
-  margin-left: 50px;
-  margin-bottom: 50px;
-  flex-direction: column;
+  display: grid;
+  grid-template-columns: repeat(8, minmax(10px,auto));
+  grid-template-rows: repeat(6, minmax(10px,auto));
   color: #c4c7cf;
+  font-size: 14px;
+`
+const Wrapper = styled.div`
+  margin-left: 45px;
+  margin-right: 45px;
+  padding-left: 100px;
+  padding-right: 100px;
+  text-size-adjust: 100%;
 `
-
 const Humble = styled.div`
+  display: block;
+  grid-column-end: 3;
+  grid-column-start: 1;
+  grid-row-end: 2;
+  grid-row-start: 1;
   margin-top: 25px;
   img {
     width: 99px;
     height: 29px;
   }
 `
-const FooterEntry = styled.div`
-  margin-top: 25px;
-  font-size: 12px;
-  &:hover {
-    color: white;
+const Referral = styled.div`
+  display: block;
+  grid-column-end: 3;
+  grid-column-start: 1;
+  grid-row-end: 7;
+  grid-row-start: 2;
+  margin-top: 10px;
+  .referral-link {
+    color: #ef9841;
   }
 `
-const Invite = styled.p`
-  color: #ef9841;
-  font-weight: bold;
-  font-size: 12px;
+const FooterNav = styled.section`
+  display: flex;
+  color: #c4c7cf;
+  flex-direction: flex;

```

Comment:
There is no flex-direction: flex

Reply:
Thank you for that catch haha, didnt mean to type that, will fix asap